src/strategies/dimo-erc20-votes-vesting/index.ts

```diff
@@ -0,0 +1,51 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { strategy as erc20VotesStrategy } from '../erc20-votes';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'RedDuck-Software';
+export const version = '0.1.0';
+
+const abi = [
+    'function balanceOf(address user) external view returns (uint256)'
+];
+
+export async function strategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+): Promise<Record<string, number>> {
+    const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+    const erc20votesScore = await erc20VotesStrategy(
+        space,
+        network,
+        provider,
+        addresses,
+        options,
+        snapshot
+    );
+
+    const multi = new Multicaller(network, provider, abi, { blockTag });
+    addresses.forEach((address) =>
+        multi.call(address, options.vesting, 'balanceOf', [address])
+    );
+
+    const vestingScore: Record<string, BigNumberish> = await multi.execute();
+
```

Comment:
Hi @kostyamospan You can use two `erc20-balance-of` strategies for this

Reply:
Hello! Where is it possible to use two strategies? Could you send a screenshot of UI, where it's located (where I can use two strategies)? I couldn't find it anywhere.

Reply:
Hi @borseno you can set this in your space settings, under strategies add two `erc20-balance-of` strategies

Reply:
![image](https://user-images.githubusercontent.com/15967809/154659240-737983be-8db6-4c5a-a66d-690824317fdb.png)